### DISCUSSION TITLE
Rich Text and Admin Perms

### DISCUSSION THREAD

**Discussion by Arinji2** [pocketbase/pocketbase#4306] [Q&A] [answered]
Hiya, so we wanted to use pocketbase for our docs website since it has a rich text editor field. 
We are using react so is there any parser available for the rich text editor? 

Also is there any way to make like an admin who can login to the pocketbase db but only be able to create new records.. basically we want it so the editors can only access the collection (called articles) and only delete or create or edit the rows inside it..

**Comment by ganigeorgiev** [comment, marked as answer]
> We are using react so is there any parser available for the rich text editor?

I don't understand what do you mean by this. The editor (TinyMCE) outputs plain HTML string.

> Also is there any way to make like an admin who can login...

No, there are no admin roles at the moment.

**Reply by gokulgseven** [reply]
(I also had a use case where I wanted to do the same things, cause making the front end for 1 user specifically was cumbersome)
I heard that the front end of the admin panel is a static site made in svelte, 
let's say i want to go about using the same code and then replace admin apis with the user apis , any suggestions as someone who made the admin Ui in the first place. 
(i actually just want to use the theme and ui elements (really bad at css ) other things i should be able to manage)

**Comment by gokulgseven** [comment]
you will have to create a normal user - with the required roles to edit the collections and create the frontend to edit it.
